README.md

```diff
@@ -2,9 +2,9 @@
 
 <p align="center">
 <a href="https://github.com/flarum/core/actions?query=workflow%3ATests"><img src="https://github.com/flarum/core/workflows/Tests/badge.svg" alt="PHP Tests"></a>
-<a href="https://packagist.org/packages/flarum/core"><img src="https://poser.pugx.org/flarum/core/d/total.svg" alt="Total Downloads"></a>
-<a href="https://packagist.org/packages/flarum/core"><img src="https://poser.pugx.org/flarum/core/v/unstable.svg" alt="Latest Version"></a>
-<a href="https://packagist.org/packages/flarum/core"><img src="https://poser.pugx.org/flarum/core/license.svg" alt="License"></a>
+<a href="https://packagist.org/packages/flarum/core"><img src="https://img.shields.io/packagist/dt/flarum/core" alt="Total Downloads"></a>
+<a href="https://packagist.org/packages/flarum/core"><img src="https://img.shields.io/packagist/v/flarum/core" alt="Latest Version"></a>
```

Comment:
This shows `v0.1.0-beta` instead of `v0.1.0-beta.13`. I'm not sure if shields can show the latest beta correctly.

Reply:
might be better to just use the lastest github release instead of packagist

![latest_version](https://img.shields.io/github/v/release/flarum/core?include_prereleases&sort=semver)

```
https://img.shields.io/github/v/release/flarum/core?include_prereleases&sort=semver
```

Reply:
@SychO9 I like that, and we can remove `include_prereleases` and it'll still work

```
https://img.shields.io/github/v/release/flarum/core?sort=semver
```

![image](https://img.shields.io/github/v/release/flarum/core?sort=semver)

Reply:
I have updated it as per the changes requested. It is now showing a full version of the latest release. Kindly review.